06-building-a-program-start-to-finish/automating-tasks/output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: Gulp
+- **Deliver**: a screencast showing gulp watching and refreshing your browser when changes are made to your HTML, CSS & JS files
+- **With**: *independently*
+
+#### Link to YouTube video
+- [Browser reload with Gulp](https://youtu.be/QW1bR0LtsdM)
```

Comment:
Moving forward, endeavour to mute your mic when recording if you won't be saying anything. There's a lot of background noise.

Other than that, good work!